static/js/components/SourceTable.jsx

```diff
@@ -482,6 +510,15 @@ const SourceTable = ({ sources, title, sourceStatus = "saved", groupID }) => {
     expandableRows: true,
     renderExpandableRow: renderPullOutRow,
     selectableRows: "none",
+    sort: false,
+    onTableChange: handleTableChange,
+    serverSide: true,
+    rowsPerPage: numPerPage,
+    page: pageNumber - 1,
+    rowsPerPageOptions: [10, 25, 50, 75, 100, 200],
+    jumpToPage: true,
+    pagination: true,
+    count: totalMatches,
```

Comment:
```suggestion
    count: totalMatches,
    filter: false,
    search: false,
```
Filtering and searching actually breaks when you enable serverSide, and you need to set up custom handlers/renders to get them to work again. I suggest we disable these for now and add that functionality back in through a future PR.

Reply:
:+1:

Reply:
Done